stacker/providers/aws/default.py

```diff
@@ -73,7 +72,10 @@ def create_policy(self):
             }""" % (self.queue_arn, self.topic_arn)
 
     def setup(self):
-        logger.debug('Creating cloudformation listener')
+        """Creates the initial SNS/SQS resources for listening to
```

Comment:
I don't see `self.topic_arn` and `self.queue_url` getting set anywhere anymore, is that ok/meant to be?